src/Servers.jl

```diff
@@ -384,7 +384,7 @@ function listenloop(f, listener, conns, tcpisvalid,
                 Base.release(sem)
             end
         catch e
-            if e.code == Base.UV_ECONNABORTED
+            if !isa(e, EOFError) && (e.code == Base.UV_ECONNABORTED)
```

Comment:
There are many other error types without a `code` field, shouldn't this check if `e` is or correct type instead? I guess `IOError`?
```suggestion
            if e isa IOError && e.code == Base.UV_ECONNABORTED
```

Reply:
Yes, makes sense